Tasks/AzureMonitorAlerts/task.json

```diff
@@ -0,0 +1,128 @@
+{
+	"id": "1d876d40-9aa7-11e7-905d-f541cc882994",
+	"name": "AzureMonitorAlerts",
+	"friendlyName": "Azure Monitor Alerts",
+	"description": "Add alert rules for Azure resources in a resource group",
+	"author": "Microsoft Corporation",
+	"helpMarkDown": "More Information -- Get the link ",
+	"category": "Deploy",
+	"visibility": [
+		"Build",
+		"Release"
+	],
+	"demands": [],
+	"version": {
```

Comment:
Should we start Versioning from 1.0.0?

Reply:
I think 0.0.1 is fine . Other new tasks, like azurevmss, or azuremonitor also start out with 0.*

Reply:
Sticking with 0.0.1